CHANGELOG.md

```diff
@@ -6,9 +6,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0] - 2022-10-03
+### Changed
+- retry support ([#27](https://github.com/MetaMask/json-rpc-middleware-stream/pull/27))
+- Q3 2022 repository standardization ([#25](https://github.com/MetaMask/json-rpc-middleware-stream/pull/25))
+- Bump path-parse from 1.0.6 to 1.0.7 ([#20](https://github.com/MetaMask/json-rpc-middleware-stream/pull/20))
+- Bump glob-parent from 5.1.1 to 5.1.2 ([#19](https://github.com/MetaMask/json-rpc-middleware-stream/pull/19))
+- Bump hosted-git-info from 2.8.8 to 2.8.9 ([#17](https://github.com/MetaMask/json-rpc-middleware-stream/pull/17))
+- Bump lodash from 4.17.19 to 4.17.21 ([#16](https://github.com/MetaMask/json-rpc-middleware-stream/pull/16))
+- Repo standardization ([#15](https://github.com/MetaMask/json-rpc-middleware-stream/pull/15))
+- Fix package.json, CI config ([#14](https://github.com/MetaMask/json-rpc-middleware-stream/pull/14))
+- Add codeowners ([#12](https://github.com/MetaMask/json-rpc-middleware-stream/pull/12))
```

Comment:
What do you think about removing the development-related changelog entries? I think it would also be a good idea to reword some of these to reveal more of the context behind the changes. Perhaps something like this?

``` suggestion
### Changed
- Retry pending requests when notification to reconnect is received ([#27](https://github.com/MetaMask/json-rpc-middleware-stream/pull/27))

### Security
- Add `@lavamoat/allow-scripts` to make dependency install scripts opt-in ([#25](https://github.com/MetaMask/json-rpc-middleware-stream/pull/25))
```